Kickstarter-iOS/Views/Storyboards/Settings.storyboard

```diff
@@ -225,16 +225,16 @@
                                                 </subviews>
                                                 <color key="backgroundColor" red="0.96862745098039216" green="0.96862745098039216" blue="0.96470588235294119" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                                 <constraints>
-                                                    <constraint firstAttribute="trailing" secondItem="IjC-cR-RB2" secondAttribute="trailing" constant="12" id="1U9-Tf-dMw"/>
-                                                    <constraint firstAttribute="bottom" secondItem="IjC-cR-RB2" secondAttribute="bottom" constant="18" id="7IG-eC-1Nw"/>
+                                                    <constraint firstItem="IjC-cR-RB2" firstAttribute="trailing" secondItem="Cdr-bL-P2W" secondAttribute="trailing" constant="-12" id="1U9-Tf-dMw"/>
+                                                    <constraint firstItem="IjC-cR-RB2" firstAttribute="bottom" secondItem="Cdr-bL-P2W" secondAttribute="bottom" constant="-18" id="7IG-eC-1Nw"/>
                                                     <constraint firstItem="IjC-cR-RB2" firstAttribute="leading" secondItem="Cdr-bL-P2W" secondAttribute="leading" constant="12" id="RcF-Rf-Ixj"/>
                                                     <constraint firstItem="IjC-cR-RB2" firstAttribute="top" secondItem="Cdr-bL-P2W" secondAttribute="top" constant="6" id="lg9-uS-DXy"/>
                                                 </constraints>
                                             </view>
-                                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ioz-I8-SoZ" userLabel="Resend Verification Email Button View">
+                                            <view hidden="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ioz-I8-SoZ" userLabel="Resend Verification Email Button View">
```

Comment:
We should probably see these explicitly in code ... as discussed in https://github.com/kickstarter/ios-oss/pull/488#discussion_r235212773

Reply:
This actually is being explicitly set [via the view model](https://github.com/kickstarter/ios-oss/blob/2278546279f5360a350f095adc75a20b39f0419e/Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift#L129) - I can remove this flag in IB.